tests/unit/conftest.py

```diff
@@ -23,3 +25,39 @@ def configure_logging():
         root_logger.removeHandler(handler)
 
     start_logging(log_dir_name="csvcubed_testing", selected_logging_level=logging.DEBUG)
+
+
+@pytest.fixture()
+def dummy_mapped_url():
+    """
+    This fixture is used to enable some tests to pass "bad input" URLs without causing an error
+    due to a corresponding local file path not existing. It maps the URLs used in those tests to
+    a file path that is known to exist. This allows connection errors and other such exceptions
+    to happen in a testing scenario.
+    """
+    # Add test URL to dictionary when ready to use fixture
+    test_dictionary = {
+        "//thisinputurlwillcauseanerror.com": APP_ROOT_DIR_PATH
```

Comment:
Could you stick comments here explaining the expected HTTP request behaviour when each of the following URLs are requested?